### DISCUSSION TITLE
Use cosmos db

### DISCUSSION THREAD

**Discussion by haywoodIII** [microsoft/dicom-server#3263] [Ideas]
Would be beautiful to have path to get structure api responses faster. currently the response study structure call performance is  sluggish for viewer integrations. Since viewers must retrieve this data before it can start painting anything interesting on the screen, it should be as fast as possible. 

Ideally, imo, ya'll should be striving for < 1 second on the metadata calls via wado-rs for the azure health data service product. This is on-par or slightly better than a industry standard like Ambra.

**Comment by arunmk-ms** [comment]
Thank you for your input. Our current storage solution is Azure storage, but I will share your idea with the team.
